src/main/java/org/spongepowered/common/event/tracking/phase/packet/PlaceBlockPacketState.java

```diff
@@ -96,7 +96,6 @@ public void appendNotifierToBlockEvent(BasicPacketContext context, IMixinWorldSe
         final Location<World> location = new Location<>(player.getWorld(), pos.getX(), pos.getY(), pos.getZ());
         final LocatableBlock locatableBlock = LocatableBlock.builder()
                 .location(location)
```

Comment:
It doesn't make sense that you're changing the builders usage in the file change above but not changing it here. Why?